release/tune_tests/cloud_tests/workloads/run_cloud_test.py

```diff
@@ -320,7 +320,7 @@ def run_tune_script_for_time(
         # Wait until indicator file exists
         wait_for_run_or_raise(process, indicator_file=indicator_file, timeout=30)
         # Stop experiment (with checkpoint) after some time
-        send_signal_after_wait(process, signal=signal.SIGINT, wait=run_time)
```

Comment:
Line 320 already waits till "indicator file exists". Is the flakiness here caused by some other states that are written after the indicator file step?

Reply:
The flakiness is mostly caused by a stale experiment state on the cloud. I.e. the local experiment checkpoint is aware of the newest checkpoints from each trial, but it waits for a prior sync to cloud to finish, and because it is forcefully terminated (instead of gracefully) this last sync may not happen. Then there is stale information on the cloud storage and the test fails